src/main/c/Objects/pyjclass.c

```diff
@@ -449,6 +462,8 @@ PyMethodDef pyjclass_methods[] = {
 
 
 static PyMemberDef pyjclass_members[] = {
+    /* TODO Consider moving to managed dict as described in the documentation for tp_dictoffset */
+    {"__dictoffset__", T_PYSSIZET, offsetof(PyJClassObject, attr), READONLY},
```

Comment:
What does this change do?

Reply:
This is the equivalent of [tp_dictoffset](https://docs.python.org/3/c-api/typeobj.html#c.PyTypeObject.tp_dictoffset) which was used previously. The documentation for [PyType_Slot](https://docs.python.org/3/c-api/type.html#c.PyType_Slot.slot) mentions that no slot is provided for tp_dictoffset and recommends switching to a managed dict, but I couldn't get that working I so followed the fallback recommendation of putting the offset in the members.